### ISSUE TITLE
Run bit install when creating workspace

### ISSUE THREAD

**Issue by debs-obrien** [teambit/bit#4684]
### Description
workspace with `bit new react` gets generated with error for env as it is missing dependencies. When you then create a component there are more errors as it can't find testing library and you have to manually run `bit install` to fix it. 

### Describe the solution you'd like
bit install should run in the process of the creation of workspace or when running bit start

**Comment by GiladShoham** [comment]
That's really weird. I'm pretty sure it's running on bit new

**Comment by davidfirst** [comment]
Yes. `bit install` is running during bit-new. It's happening in the last step, after importing the components and compiling them.
@debs-obrien , are you referring to this error during bit-new?
```
✖ error: unable to load the extension "company.scope/templates/envs/my-react", please use the '--log' flag for the full error.
```
I have tried to fix it for a few hours, but it's not that easy. I'll describe the complexity later. Before, I want to make sure that you don't see any other issue. I did try to run `bit status` after `bit new` and it works fine with no errors.
